src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -156,7 +304,13 @@ private static void SetPreciseMachineCounters(IEnumerable<ProviderInfo> provider
                 }
 
                 if (profileSourceIDs.Count > 0)
+                {
+                    // 
+                    // FIXME: This function changes the -pmcsources intervals machine wide.
```

Comment:
[https://github.com/Microsoft/xunit-performance/issues/192](https://github.com/Microsoft/xunit-performance/issues/192)

---
In reply to: [102348741](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r102348741) [](ancestors = 102348741,102335560)